src/lib/modules/SearchControls/overridden/CardBucketAggregationValueElement.js

```diff
@@ -2,6 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Checkbox, Label, List } from 'semantic-ui-react';
 
+const generateLabel = key => {
```

Comment:
👍 
I have the feeling that there are other filters cmps, to apply the same fix maybe?

Reply:
After checking, the [overriden](https://github.com/inveniosoftware/react-invenio-app-ils/tree/master/src/lib/modules/SearchControls/overridden) folder, there are no more labels that need to be generate ( in `AvailableLoanBucketAggregationValues.js` we use a static [label](https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/modules/SearchControls/overridden/AvailableLoanBucketAggregationValues.js#L16) )